portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,45 @@
+
+package com.google.sps.servlets;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/comments")
+public class CommentServlet extends HttpServlet {
+
+    private CommentSection section = new CommentSection();
+
+    @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+    String json = new Gson().toJson(section);
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the input from the form.
+    int input = getContent(request);
+
+    section.takePlayerTurn(content);
```

Comment:
You should probably rename this method

Reply:
It seems I didn't push all the changes when I made the pull request, but I'm resolving because this is now fixed.